docs/quickstart.md

```diff
@@ -15,6 +15,16 @@ with the only help of an Elemental Teal iso
 
 ---
 
+### What is the Rancher Elemental Stack ?
+
+The Elemental Stack consists of some packages on top of SLE Micro for Rancher
+
+- **elemental-toolkit** - includes a set of OS utilities to enable OS management via containers. Includes dracut modules, bootloader configuration, cloud-init style configuration services, etc.
+- **elemental-operator** - this connects to Rancher Manager and handles machineRegistration and machineInventory CRDs
+- **elemental-register** - this registers machines via machineRegistrations and installs them via elemental-cli
+- **elemental-cli** - this installs any elemental-toolkit based derivative. Basically an installer based on our A/B install and upgrade system
+- **rancher-system-agent** - runs on the installed system and gets instructions ("Plans") from Rancher Manager what to install and run on the system
```

Comment:
we all wished that things would be _that_ simple.
In reality, we run `elemental-system-agent` first which only purpose is a single plan: Download an start `rancher-system-agent` 🤦🏻‍♂️

Reply:
Changed to `elemental-system-agent`. As this is the quickstart, I'm hesitant to make the distinction here.